CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 Dates are in `dd-mm-yyyy` format.
 
+## [2.1.1] - 23-01-2023
+
+### Changed
+
+- Allow xpub exports at the `m/45'/coin_type'/account'` derivation paths.
+- Allow up to 8 derivation steps for BIP-32 paths (instead of 6).
```

Comment:
Maybe you could add that requirements for address index were loosen and checks for change address to be either 1 or 0 were removed :)

Reply:
Added a more generic explanation.